docs/README.md

```diff
@@ -0,0 +1,75 @@
+# WIC Montana Mock API
+
+# Introduction
+This is the API layer of the WIC Montana Mock API. It includes a few separate components:
+- The REST API
+- Backend & utility scripts
+
+## API Directory Structure
+```
+.
+├── api
+│   └── auth                Authentication code for API
+│   └── db
+│       └── models          DB model definitions            
+│       └── migrations      DB migration configs
+│           └── versions    The DB migrations
+│   └── logging
+│   └── route               API route definitions
+│       └── handler         API route implementations
+│   └── scripts             Backend scripts that run separate from the application
+│   └── util                Utility methods and classes useful to most areas of the code
+│
+├── tests
+├── Dockerfile          Multi-stage Docker build file for project
+├── docker-compose.yml  Config file for docker-compose tool, used for local development
+├── local.env           Environment variable configuration for local files
+├── openapi.yml         API specification
+├── pyproject.toml      Python project configuration file
+└── setup.cfg           Python config for tools that don't support pyproject.toml yet
+```
+
+# Getting started
+
+1. In your terminal, `cd` to this repo.
+2. Make sure you have [Docker Desktop](https://www.docker.com/products/docker-desktop/) installed & running.
+3. Run `make init start` to build the image and start the container.
+4. Navigate to `localhost:8080/v1/docs` to access the Swagger UI.
+5. Run `make run-logs` to see the logs of the running API container
+6. Run `docker-compose down` when you are done to delete the container.
+
+# Information
+
+* [API Technical Overview](/docs/app/technical-overview.md)
+* [Database Migrations](/docs/app/database-migrations.md)
+* [Formatting and Linting](/docs/app/formatting-and-linting.md)
+* [Writing Tests](/docs/app/writing-tests.md)
+
+## Additional Useful Commands
```

Comment:
```suggestion
## Some Useful Commands
```
Can we add a line or two explaining why we chose the strategy of using make?

Reply:
Hmm, I'm actually unsure of how to answer that. "Unfamiliar with alternatives / that's the way the prior approach did it" don't seem like the right answer.